contracts/stablecoinswap.vy

```diff
@@ -128,6 +128,29 @@ def removeLiquidity(token_address: address, stableswap_token_amount: uint256, er
 
     return True
 
+@public
+@constant
+def tokenExchangeRateAfterFees(input_token_address: address, output_token_address: address) -> uint256:
+    assert self.inputTokens[input_token_address] and self.outputTokens[output_token_address]
```

Comment:
We already have the assert in swapTokens. I think we can remove it from here and from tokenOutputAmountAfterFees to save gas.